pkg/registry/core/pod/strategy.go

```diff
@@ -644,9 +644,9 @@ func applySeccompVersionSkew(pod *api.Pod) {
 		})
 }
 
-// seccompFieldForAnnotation takes a pod seccomp profile field and returns the
+// SeccompAnnotationForField takes a pod seccomp profile field and returns the
 // converted annotation value
-func seccompAnnotationForField(field *api.SeccompProfile) string {
+func SeccompAnnotationForField(field *api.SeccompProfile) string {
```

Comment:
Moving this function to a location like https://github.com/kubernetes/kubernetes/blob/master/pkg/api/pod/util.go might simplify the package dependency graph. Run the import verification scripts to make sure our import restrictions are up to date so verify CI will pass.

Reply:
Thanks for the hint, moved it into the pod utils and updated the import paths.